airbyte-integrations/connectors/destination-redshift/src/test-integration/java/io/airbyte/integrations/destination/redshift/SshRedshiftDestinationBaseAcceptanceTest.java

```diff
@@ -201,8 +217,11 @@ protected void tearDown(final TestDestinationEnv testEnv, HashSet<String> TEST_S
         JdbcUtils.HOST_LIST_KEY,
         JdbcUtils.PORT_LIST_KEY,
         config -> {
-          getDatabaseFromConfig(config).query(ctx -> ctx.fetch(String.format("DROP USER IF EXISTS %s;", USER_WITHOUT_CREDS)));
+          getDatabase().query(ctx -> ctx.fetch(String.format("DROP USER IF EXISTS %s;", USER_WITHOUT_CREDS)));
         });
+    connection.setAutoCommit(false);
+    connection.commit();
+    connection.close();
```

Comment:
+1

Reply:
Can these 3 commands be wrapped in another method so we don't forget to do them all?

Reply:
Done